moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -70,14 +70,14 @@ namespace moveit
 {
 namespace planning_interface
 {
+namespace
+{
 const std::string MoveGroupInterface::ROBOT_DESCRIPTION =
     "robot_description";  // name of the robot description (a param name, so it can be changed externally)
 
 const std::string GRASP_PLANNING_SERVICE_NAME = "plan_grasps";  // name of the service that can be used to plan grasps
 const rclcpp::Logger LOGGER = rclcpp::get_logger("move_group_interface");
 
```

Comment:
```suggestion
static const std::string MoveGroupInterface::ROBOT_DESCRIPTION =
    "robot_description";  // name of the robot description (a param name, so it can be changed externally)

const std::string GRASP_PLANNING_SERVICE_NAME = "plan_grasps";  // name of the service that can be used to plan grasps
const rclcpp::Logger LOGGER = rclcpp::get_logger("move_group_interface");

namespace
{
```